test/data/addReactions_err_test.yaml

```diff
@@ -0,0 +1,169 @@
+description: |-
+  This is the file for testing Plog errors
+
+generator: ck2yaml
+input-files: [mech.inp, therm.DAT, trans.dat]
+cantera-version: 2.5.0a2
+date: Mon, 29 Jun 2020 14:13:58 +0000
+
```

Comment:
I would suggest deleting these lines to keep the test file as small as possible.